package.json

```diff
@@ -94,7 +94,7 @@
       "dot-location": [2, "property"],
       "no-implied-eval": 2,
       "no-loop-func": 2,
-      "no-magic-numbers": 1,
+      "no-magic-numbers": [1, {"ignore": [-1, 0, 1, 2]}],
```

Comment:
I feel like it doesn't make sense to always document/explain all those numbers -
 - `-1` usually is for negative indexOf
 - 0, 1, 2 are usually for first character in the string, or tuple elements.